seam/utils/options.py

```diff
@@ -0,0 +1,42 @@
+from typing import Optional
+
+
+class SeamHttpInvalidOptionsError(Exception):
```

Comment:
This question applies to all custom exceptions (like the action attempt ones added earlier).

Do we need to export the exceptions to enable users to match on the, e.g. with raise/catch blocks?

Reply:
c9e20ecb2f858002764c534ac210871f6e053022